File: script/python/cmd_line/CmdLine.py

def on_cmd(cmd):
    cmd_arr = cmd.split(" ")
    if len(cmd_arr) == 0:
        return
    cmd = cmd_arr[0]
    if cmd == "hotfix":
        import hotfix.hotfix
        hotfix.hotfix.start_hotfix()
    else:
        print("unknown cmd:", cmd)

File: script/lua/base/cmd_line.lua

require("util.str_util")
require("util.table_util")
require("base.hotfix")

cmdLine = {}

function cmdLine.on_cmd(args)
    print("on_cmd", args)
    local cmd_arr = StrUtil.split(args, " ")
    if TableUtil.isEmpty(cmd_arr) then return end
    local cmd = cmd_arr[1]
    if cmd == "hotfix" then
        Hotfix.hotfix()
    else
        print("unknown cmd:", cmd)
    end
end
